packages/cli/test/integration/owner.api.test.ts

```diff
@@ -1,30 +1,36 @@
 import express from 'express';
+import type { SuperAgentTest } from 'supertest';
 import validator from 'validator';
 
-import * as utils from './shared/utils';
-import * as testDb from './shared/testDb';
-import { Db } from '../../src';
 import config from '../../config';
+import { Db } from '../../src';
```

Comment:
Out of scope: For consistency, in future we should find a way for all DB calls in tests to be done through `testDb`.

Reply:
maybe a tech debt ticket?

Reply:
https://linear.app/n8n/issue/N8N-4147/centralize-db-calls-in-tests